lib/sass/script/functions.rb

```diff
@@ -1670,6 +1687,42 @@ def feature_exists(feature)
     end
     declare :feature_exists, [:feature]
 
+    # Returns a reference to a function for later invocation with the `call` function.
+    #
+    # The function reference created may refer to a function defined in
+    # your stylesheet, built-in to the host environment, or a CSS native
+    # function. As such, this function never returns an error, but the
+    # you can use `function-exists()` to check if the function reference
+    # created points at a Sass function.
```

Comment:
Decision: `get-function($name, $css: true)` will always return a css reference even if the sass function exists. `get-function($name)` will always return a sass reference or an error.